README.md

```diff
@@ -31,22 +31,38 @@ cd osparc-ops
 make help
 ```
 
+### WSL deployement
+
+If you are using WSL1, clone the repository under the Windows filesystem and access it in WSL. You need to ensure [Ensure Volume Mounts Work](https://nickjanetakis.com/blog/setting-up-docker-for-windows-and-wsl-to-work-flawlessly) (Section Volume Mounts Work)
```

Comment:
actually, maybe you should change to something like:
clone the repository in a path accessible by the Windows filesystem (typically in /mnt/c/my_path and not in /home)

If you clone using windows git, then you might get into the problem that the git in windows and wsl are not in sync...

Reply:
I don't understand your last sentence. What would not be in sync ? Are they not reading the sames files in the folder .git ?

Reply:
Ah ok maybe there is a misunderstanding here. If it's important to clone it in the windows filesystem, this is to ensure volume will be mount (they are not accessible by the windows docker engine when the folder of the project is /home)

Reply:
ok so just for your info...
using WSL2 it is important to do the exact contrary... so I would remove this stuff or enhance.